static/views/popup.html

```diff
@@ -12,8 +12,8 @@
   </head>
 
   <body
-    class="w-96 overflow-x-hidden bg-gray-50 dark:bg-surface-00dp slashed-zero"
-    style="height: 600px"
+    class="w-96 overflow-x-hidden bg-gray-50 min-h-screen dark:bg-surface-00dp slashed-zero"
+    style="max-width: 100%; height: 600px"
```

Comment:
Why max-width? Shouldn't the width of the body just be 100%? (so it always fills the screen width and serve as a container for components?

Reply:
using just width has side effect over w-96 and on desktop version it decrease popup width. this change makes responsive on mobile without effecting desktop app designs.